tools/build_config/build.sh

```diff
@@ -173,7 +174,10 @@ function main() {
   fi
 
   if ((build)); then
-    run_build "${build_dir}"
+      if [ ! -e ${build_dir}/CMakeCache.txt ]; then
```

Comment:
is there a way the boolean test here (`build && !-e CMakeCache.txt`) could be combined with the preceding `if ((configure))` test so the non-trivial `run_configure` call isn't duplicated?

Reply:
Realistically, we could just replace `((configure))` with `if [ ! -e ${build_dir}/CMakeCache.txt ]; then`, I guess, because you can't do anything without the configure.

Reply:
Removing the 'configure' flag would prevent the regeneration of configuration if anything were changed. To avoid that we'd need to add a 'clean' option to purge the build folder and CMake configuration.